SPECS/yasm/yasm.spec

```diff
@@ -0,0 +1,230 @@
+Summary:        Modular Assembler
+Name:           yasm
+Version:        1.3.0
+Release:        13%{?dist}
+License:        BSD and (GPLv2+ or Artistic or LGPLv2+) and LGPLv2
+URL:            https://yasm.tortall.net/
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        http://www.tortall.net/projects/%{name}/releases/%{name}-%{version}.tar.gz
```

Comment:
>http://www.tortall.net/projects/%{name}/releases/%{name}-%{version}.tar.gz [](start = 16, length = 74)

Please switch to the HTTPS version, since it works on this site

Reply:
Done